app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/ContributorsActivity.java

```diff
@@ -51,8 +54,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.item_repository:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017"));
-                startActivity(intent);
+                navigator.navigateToWebPage("https://github.com/DroidKaigi/conference-app-2017");
```

Comment:
@konifar Sorry, please tell me 🙇 
How should I call the method of ViewModel that Fragment has from `ContributorsActivity#onOptionsItemSelected()` method?
In my opinion, I also create `onClickRepositoryMenu()` methods in Fragment , and instantiate Fragment in `ContributorsActivity#onCreate()`.
I might have made a mistake... 🤔